console/console.go

```diff
@@ -95,7 +95,7 @@ func GetConsolePluginCR(pluginName string, displayName string, consolePort int,
 //+kubebuilder:rbac:groups="",resources=services,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=console.openshift.io,resources=consoleplugins,verbs=*
 
-func InitConsole(client client.Client, odfPort int, ibmPort int) error {
+func InitConsole(client client.Client, consolePort int) error {
```

Comment:
keep the name as it was before to minimize the changes.

Reply:
The names don't make much sense. I think it's better to make variable names contextually correct.

Reply:
If thats a case lets do it at all the places even in main.go `odfConsolePort` should be `ConsolePort`

Reply:
Even if the names don't make much sense, the renaming should be in a separate commit.

* one commit: remove ibm console
* one or more separate commits: do follow-up renames and cleanups